gulpfile.js

```diff
@@ -93,14 +93,20 @@ function deployWebpack() {
         .on('error', logError);
 }
 
+function installNpmDependencies() {
+    return gulp.src('package.json')
+        .pipe(exec('npm install'), logError)
```

Comment:
Does it make sense to check if the version of npm supports `ci` and if there's also a package-lock.json file, to use `npm ci` instead?

Reply:
If we use `npm install` at build time, we run the risk of making minor version changes at build time that were not validated by the author.

If we can't use `npm ci` to update the package version in `package-lock.json` (which would ensure that only the version number is updated and no dependencies), then I think we should modify the bump task to bump the version number in both `package.json` and `package-lock.json`.